kratos/utilities/divide_triangle_2d_3.cpp

```diff
@@ -147,10 +147,7 @@ namespace Kratos
                 TriangleGetNewConnectivityGID(idivision, t.data(), mSplitEdges.data(), &i0, &i1, &i2);
 
                 // Generate a pointer to an auxiliar triangular geometry made with the subdivision points
-                IndexedPointGeometryPointerType p_aux_partition = Kratos::make_shared<IndexedPointTriangleType>(
-                    mAuxPointsContainer(i0),
-                    mAuxPointsContainer(i1),
-                    mAuxPointsContainer(i2));
+                IndexedPointGeometryPointerType p_aux_partition = GenerateTriangle(i0, i1, i2);
```

Comment:
```suggestion
                IndexedPointGeometryPointerType p_aux_partition = GenerateAuxiliaryPartitionTriangle(i0, i1, i2);
```
To make the method name more self-descriptive.